src/gen3-ui-component/components/filters/RangeFilter/UnitCalculator/UnitCalculator.css

```diff
@@ -0,0 +1,81 @@
+.mini-app {
+  background-color: var(--g3-color__bg-cloud);
+  padding: 30px;
+  margin-top: 20px;
+}
+
+.p-spacing {
+  margin-bottom: 10px;
+}
+
+.button-exit {
+  border: 1px solid var(--pcdc-color__primary);
+  background-color: var(--g3-color__white);
+  height: 25px;
+  width: 25px;
+  float: right;
+  text-align: center;
+}
+
+.container {
+  width: 73%;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  padding-bottom: 10px;
+}
+
+.group-inline {
+  display: flex;
+  align-items: center;
+}
+
+.group-inline label {
+  width: 60px;
+  font-weight: 500;
+}
+
+.full-width {
+  width: 100%;
+  padding: 5px;
+  box-sizing: border-box;
+  margin-left: 10px;
+}
+
+input,
+select {
+  font-family: inherit;
+  color: inherit;
+}
+
+.button-group-inline {
+  display: flex;
+  align-items: center;
+}
+
+.button-assign {
+  display: inline-block;
+  width: 100px;
+  background-color: var(--g3-color__white);
+  color: inherit;
+  font-family: inherit;
+  margin-top: 10px;
```

Comment:
style nit: possibly add a bottom margin here? There isn't much space under the text in the assign button judging by the screenshot